personalities/sctfe/mockstorage/gen.go

```diff
@@ -0,0 +1,18 @@
+// Copyright 2016 Google LLC. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package mockclient provides a mockable version of the Trillian log client API.
+package mockstorage
+
+//go:generate mockgen -package mockstorage -destination mock_ct_storage.go github.com/transparency-dev/trillian-tessera/personalities/sctfe Storage
```

Comment:
Could do with a README in the personalities folder or somewhere with instructions about `go gen` and tooling. It can come in another PR. Consider something like I did for Trillian where there's a docker image with the correct versions of tooling, and instructions on how to regenerate the files.